modules/lib/src/main/resources/assets/js/app/inputtype/selector/ui/NewContentButtonUploaderHandler.ts

```diff
@@ -0,0 +1,127 @@
+import {ProgressBar} from '@enonic/lib-admin-ui/ui/ProgressBar';
+import {UploadItem} from '@enonic/lib-admin-ui/ui/uploader/UploadItem';
+import {Content} from '../../../content/Content';
+import {Element} from '@enonic/lib-admin-ui/dom/Element';
+import {FormItemEl} from '@enonic/lib-admin-ui/dom/FormItemEl';
+
+export class NewContentButtonUploaderHandler {
```

Comment:
This class is quite generic and its main purpose is to visualise upload progress (not necessarily for new content and not necessarily inside a button), so it should be renamed to something like "`UploadProgressBar`". I would also make it even more generic by replacing `Content` type with generic `<T>` (`export class UploadProgressBar<T>`) but it's not important right now.